systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1223,7 +1224,9 @@ void testKMM2RollAfterSecretsCertsUpdateTLS() {
 
         LOGGER.info("Consuming messages in target cluster: {}/{}", testStorage.getNamespaceName(), testStorage.getTargetClusterName());
         resourceManager.createResourceWithWait(targetClients.consumerTlsStrimzi(testStorage.getTargetClusterName()));
-        ClientUtils.waitForInstantConsumerClientSuccess(testStorage);
+        // Extend the timeout for clients to be sure that all messages are synced by MM2
+        JobUtils.waitForJobSuccess(testStorage.getConsumerName(), testStorage.getNamespaceName(), TestConstants.GLOBAL_TIMEOUT_LONG);
+        JobUtils.deleteJobsWithWait(testStorage.getNamespaceName(), testStorage.getConsumerName());
```

Comment:
Isn't 10 minutes little bit overkill?
Also, I think that having a possibility to specify the timeout for the clients in the method would be better than splitting it like this.

Reply:
I wanted to avoid changed whole clients API just because of this. Clients API is very limited from my POV and I wanted to avoid adding another 200 lines just  because of this. 

Regarding the timeout....I didn't find any other solution as the mm2 config should force topic sync for mm2 every 10s.